preliz/maxent.py

```diff
@@ -40,6 +40,34 @@ def maxent(
     -------
 
     axes: matplotlib axes
+
+    See Also
+    --------
+    quartile : Find the distribution with the specified quartiles.
+
+    Examples
+    --------
+    Calculate the maxent Gamma distribution with 90 % of the mass between 1 and 8:
+
+    .. plot::
+        :context: close-figs
```

Comment:
```suggestion
        :context: close-figs
        :include-source: true
```

I think given the situation you describe, the best option is to keep the default to false and add this to the code examples you want the source to be shown.